de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/repository/ServerRepositoryTest.java

```diff
@@ -197,6 +210,66 @@ public void testSyncProject() throws Exception {
 				containsString(VirSatGitVersionControlBackend.BACKEND_REPOSITORY_COMMIT_PULL_MESSAGE));
 	}
 	
+	@Test
+	public void testSyncRepositoryModelChanges() throws Exception {
+		ServerRepository testServerRepository = new ServerRepository(localRepoHome, testRepoConfig);
+		testServerRepository.checkoutRepository();
+		
```

Comment:
Test case coud use some short description at the beginning outlining shortly the test process.